### PR TITLE
 pythonPackages.dateparser: 0.3.2-pre-2016-01-21 -> 0.6.0 

### PR DESCRIPTION
**Pull request by makefu**
###### Motivation for this change

unbreak pythonPackages.dateparser, bump to latest rev. This PR is part of the testing for https://github.com/makefu/nix-src-updater .

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [NA] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [NA] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/development/python-modules/dateparser/default.nix [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+, umalqurra
+, pytz
+, tzlocal
+, regex
+, ruamel_yaml }:
+buildPythonPackage rec {
+  name = "dateparser-${version}";
+  version = "0.6.0";
+
+  src = fetchgit {
```
Comment (FRidh) [inline pkgs/development/python-modules/dateparser/default.nix]: fetchFromGitHub

File: pkgs/development/python-modules/dateparser/default.nix [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    rev = "refs/tags/v${version}";
+    sha256 = "0q2vyzvlj46r6pr0s6m1a0md1cpg9nv1n3xw286l4x2cc7fj2g3y";
+  };
+
+  LC_ALL = "en_US.UTF-8";
+
+  # Upstream Issue: https://github.com/scrapinghub/dateparser/issues/364
+  disabled = isPy3k;
+
+  buildInputs = [ nose nose-parameterized mock glibcLocales ];
```
Comment (FRidh) [inline pkgs/development/python-modules/dateparser/default.nix]: test inputs go in `checkInputs`

File: pkgs/development/python-modules/dateparser/default.nix [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+  name = "dateparser-${version}";
+  version = "0.6.0";
+
+  src = fetchgit {
+    url = "https://github.com/scrapinghub/dateparser.git";
+    rev = "refs/tags/v${version}";
+    sha256 = "0q2vyzvlj46r6pr0s6m1a0md1cpg9nv1n3xw286l4x2cc7fj2g3y";
+  };
+
+  LC_ALL = "en_US.UTF-8";
```
Comment (FRidh) [inline pkgs/development/python-modules/dateparser/default.nix]: likely only needed for tests -> `export` in `preCheck`

File: pkgs/development/python-modules/dateparser/default.nix [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+    # skip because of missing convertdate module, which is an extra requirement
+    rm tests/test_jalali.py
+  '';
+
+  propagatedBuildInputs = [ six jdatetime pyyaml dateutil
+            umalqurra pytz tzlocal regex ruamel_yaml ];
+
+  meta = with stdenv.lib;{
+    description = "Date parsing library designed to parse dates from HTML pages";
+    homepage = https://pypi.python.org/pypi/dateparser;
```
Comment (FRidh) [inline pkgs/development/python-modules/dateparser/default.nix]: upstream homepage is https://github.com/scrapinghub/dateparser

**Review by FRidh — CHANGES_REQUESTED** [review]

**Comment by makefu** [comment]
This is what happens when you try to bump a version of some obscure python package 💃 
I will fix this after lunch

File: pkgs/development/python-modules/dateparser/default.nix [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+, jdatetime
+, pyyaml
+, dateutil
+, umalqurra
+, pytz
+, tzlocal
+, regex
+, ruamel_yaml }:
+buildPythonPackage rec {
+  name = "dateparser-${version}";
```
Comment (FRidh) [inline pkgs/development/python-modules/dateparser/default.nix]: Just one more tiny change.
```
pname = "dateparser";
version = "0.6.0";
```
and drop `name`.

Comment (makefu) [reply pkgs/development/python-modules/dateparser/default.nix]: ah, that works now! I remember that name was always necessary.

**Pull request merged**